client/src/hooks/reducers/feedReducers.js

```diff
@@ -55,60 +37,17 @@ export const optionsReducer = (oldState, action) => {
   }
 };
 
-export const postsReducer = (state = postsState, action) => {
+export const deletePostState = {
+  deleteModalVisibility: DELETE_MODAL_HIDE,
+};
+
+export const deletePostModalreducer = (state = deletePostState, action) => {
```

Comment:
If we're going to reorganizae this `postReducer` we should include `SET_DELETE_MODAL_VISIBILITY` instead of leaving behind this specific `deletePostModalreducer`